microsoft/testsuites/rust_vmm_mshv/rust_vmm_mshv_test.py

```diff
@@ -34,6 +34,12 @@ def before_case(self, log: Logger, **kwargs: Any) -> None:
                 "Rust Vmm MSHV test can be run with MSHV wrapper (/dev/mshv) only."
             )
 
+    def after_case(self, log: Logger, **kwargs: Any) -> None:
+        node = kwargs["node"]
+        dmesg = node.tools[Dmesg]
+        dmesg_log = dmesg.get_output(force_run=True)
+        log.debug(f"Dmesg Logs: {dmesg_log}")
```

Comment:
@smit-gardhariya could you please check whether get_output has print the dmesg out?

Reply:
Yes Lili, We use the same for CLH tests and it prints.

Reply:
> Yes Lili, We use the same for CLH tests and it prints.

if ```dmesg_log = dmesg.get_output(force_run=True)``` can print dmesg, can we remove line 41?

Reply:
Done